### PR TITLE
Migrate nephio-controllers from examples to core

### PR DESCRIPTION
**Pull request by johnbelamaric**
And rename to nephio-operator, update image tag to :latest.

### CODE REVIEW

File: nephio/core/nephio-operator/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# nephio
```
Comment (electrocucaracha) [inline nephio/core/nephio-operator/README.md:1]:
```suggestion
# nephio-operator
```

**Comment by electrocucaracha** [comment]
@johnbelamaric isn't redundant to call this package `nephio-operator` given that it lives under the `nephio` folder? What about just `operator` like the `nephio/core/operator` folder?

**Comment by johnbelamaric** [comment]
> At the end, users can clone it with whatever name that they prefer

True, so we could do as you suggest. My only (minor) concerns with that would be how it appears in lists (though the repo should be with it) and whether people cloning it would by default keep the same name (which is super generic).

I am ok either way, though. Anyone else have an opinion?

**Comment by electrocucaracha** [comment]
Ok, we can create a topic in the GitHub discussion to evaluate pros and cons. I think for now we can merge this one

/lgtm

**Pull request merged**